pkg/streamingpromql/operators/binary_operation_test.go

```diff
@@ -45,6 +45,10 @@ func TestBinaryOperation_SeriesMerging(t *testing.T) {
 					},
 				},
 			},
+			sourceSeriesIndices: []int{0},
+			sourceSeriesMetadata: []types.SeriesMetadata{
```

Comment:
[nit] We can skip providing `sourceSeriesMetadata` for cases that don't produce error messages - the labels are only needed to produce error messages.

Reply:
I did this for consistency.